potodo/potodo.py

```diff
@@ -173,7 +172,7 @@ def exec_potodo(
     :param counts: Render list with counts not percentage
     """
 
-    # Initialize the arguments
+    # initialize the arguments
```

Comment:
?

Reply:
Probably due to a typo or a find and replace ... dunno :] fixing it.